packages/dart/npt_flutter/lib/features/policy/models/policy.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'policy.g.dart';
+
+@JsonSerializable()
+class DaemonAtSign {
+  final String atSign;
+
+  DaemonAtSign({
+    required this.atSign,
+  });
+
+  factory DaemonAtSign.fromJson(Map<String, dynamic> json) => _$DaemonAtSignFromJson(json);
+  Map<String, dynamic> toJson() => _$DaemonAtSignToJson(this);
+}
+
+@JsonSerializable()
+class UserAtSign {
+  final String atSign;
+
+  UserAtSign({
+    required this.atSign,
+  });
+
+  factory UserAtSign.fromJson(Map<String, dynamic> json) => _$UserAtSignFromJson(json);
+  Map<String, dynamic> toJson() => _$UserAtSignToJson(this);
+}
```

Comment:
These are both single entry maps, which seems is this based on the current format?